docs/source/releases/1.14.4/1.14.4-open-source-upload.yaml

```diff
@@ -0,0 +1,7 @@
+Release Updates - Open Source Upload:
+- title: Open source upload
+  description: "This repo updated with current geoips version release"
+  files:
+    modified:
+    - standard sync files
+    - release notes
```

Comment:
Updated release process to always add this file, so if the latest tagged version is, say, 1.14.3, a new folder with 1.14.4/1.14.4-open-source-upload.yaml will be created.

If the latest tagged version was already 1.14.4, it will just add 1.14.4-open-source-upload.yaml to the already existing 1.14.4 folder.

This ensures we always have a valid 1.14.4 release note folder for the 1.14.4 release.